sdk/Lusid.Sdk.Tests/tutorials/Ibor/StructuredResultStore.cs

```diff
@@ -0,0 +1,946 @@
+                new Dictionary<string, CreateDataMapRequest> {{"dataMapKey", request}});
+
+            // Generate and upsert the document.
+            // NB: the result does _not_ match the portfolio contents, this is to demonstrate/validate that entire calculation is elided
+            string document = $"LusidInstrumentId,holding-ccy,pv,pv-ccy,UserDefinedData\n" +
+                              $"LUID_TEST0000,USD,100.0,USD,\"exampleData1\"\n" +
+                              $"LUID_TEST0001,USD,101.0,ZAR,\"exampleData2\"";
+            StructuredResultData structuredResultData = new StructuredResultData("csv", "1.0.0", documentId, document, dataMapKey);
+            StructuredResultDataId structResultDataId = new StructuredResultDataId("Client", documentId, effectiveAt, resultType);
+            var upsertDataRequest = new UpsertStructuredResultDataRequest(structResultDataId, structuredResultData);
+            _structuredResultDataApi.UpsertStructuredResultData(documentScope, new Dictionary<string, UpsertStructuredResultDataRequest>{{documentId, upsertDataRequest}});
+            
+            // Create result data key rule specifying 
+            var resultDataKeyRule = new PortfolioResultDataKeyRule("Client", documentScope, documentId,  portfolioCode : portfolioCode, portfolioScope : scope, resultKeyRuleType: ResultKeyRule.ResultKeyRuleTypeEnum.PortfolioResultDataKeyRule);
+
+            // Create and upsert a recipe with the result data key rule
+            var pricingOptions = new PricingOptions {AllowAnyInstrumentsWithSecUidToPriceOffLookup = false, AllowPartiallySuccessfulEvaluation = true};
+            var pricingContext = new PricingContext(null, null, pricingOptions, new List<ResultKeyRule>{resultDataKeyRule} );
+            var configurationRecipe = new ConfigurationRecipe(documentScope, "recipe", new MarketContext(), pricingContext);
+            var upsertRecipeRequest = new UpsertRecipeRequest(configurationRecipe, null);
+            _recipeApi.UpsertConfigurationRecipe(upsertRecipeRequest);
+            
+            // Create a valuation request, requesting LusidInstrument Id, Pv amount and Some
+            var valuationRequest = new ValuationRequest(
+                recipeId: new ResourceId(documentScope, "recipe"),
+                metrics: new List<AggregateSpec>
+                {
+                    new AggregateSpec(TestDataUtilities.Luid, AggregateSpec.OpEnum.Value),
+                    new AggregateSpec("Valuation/PV", AggregateSpec.OpEnum.Value),
+                    new AggregateSpec("UnitResult/UserDefinedData", AggregateSpec.OpEnum.Value)
+                },
+                portfolioEntityIds: new List<PortfolioEntityId>
+                {
+                    new PortfolioEntityId(scope, portfolioCode),
+                },
+                valuationSchedule: new ValuationSchedule(effectiveAt: effectiveAt));
+    
+            // Perform valuation and obtain results
```

Comment:
I think this is returning the PVs from the SRS rather than calculating from quotes store - is it worth added a comment to that effect?

Reply:
It takes the accrual from SRS into account and then calculates it using the ValuationEngine, gave this explanation at the top of the test.